meteor/sofie-server/model/entities/rundown.ts

```diff
@@ -136,8 +147,12 @@ export class Rundown {
 		return this.nextPart
 	}
 
-	public isActive(): boolean {
-		return this.isRundownActive
+	public getActiveStatus(): boolean {
+		return this.isActive
+	}
+
```

Comment:
`getActiveStatue` is a bit weird phrasing. 
What's more understandable:
```
if (rundown.isActive) { ... }
```
or 
```
if (rundown.getActiveStatus) { ... } 
```

It would be prefered that the "outwards facing" "get status" was called `isActive`. I know this clashes with the internal `isActive`, but maybe it's the internal variable that needs to change so we can get a nice "API" for people consuming our Entity.

Reply:
Perhabs we should add a prefixed character, for internal values, to avoid this clash, while still keeing a good name.
In C# private fields are commonly prefixed with '_'. 
It sounds like the current `isActive` is an private field, so we could maybe do something similar here?

Reply:
I have changed it to `isActive()`.